src/DIRAC/FrameworkSystem/Agent/ComponentSupervisionAgent.py

```diff
@@ -550,7 +552,7 @@ def _checkServiceURL(self, serviceName, options):
     system = options['System']
     module = options['Module']
     self.log.info('Checking URLs for %s/%s' % (system, module))
-    urlsConfigPath = os.path.join('/Systems', system, self.setup, 'URLs', module)
+    urlsConfigPath = PathFinder.getComponentSection(os.path.join(system, module), False, self.setup, 'URLs')
```

Comment:
what path is this going to return?

Reply:
```
In [14]: PathFinder.getComponentSection(os.path.join('DataManagement', 'FileCatalog'), False, 'LHCb-Production', 'URLs')
Out[14]: '/Systems/DataManagement/Production/URLs/FileCatalog'

In [15]: gConfig.getValue('/Systems/DataManagement/Production/URLs/FileCatalog')
Out[15]: 'dips://lbvobox302.cern.ch:9197/DataManagement/FileCatalog, dips://lbvobox303.cern.ch:9197/DataManagement/FileCatalog, dips://lbvobox304.cern.ch:9197/DataManagement/FileCatalog, dips://lbvobox305.cern.ch:9197/DataManagement/FileCatalog, dips://lbvobox306.cern.ch:9197/DataManagement/FileCatalog, dips://lbvobox307.cern.ch:9197/DataManagement/FileCatalog, dips://lbvobox308.cern.ch:9197/DataManagement/FileCatalog, dips://lbvobox309.cern.ch:9197/DataManagement/FileCatalog, dips://lbvobox310.cern.ch:9197/DataManagement/FileCatalog, dips://lbvobox311.cern.ch:9197/DataManagement/FileCatalog, dips://lbvobox312.cern.ch:9197/DataManagement/FileCatalog, dips://lbvobox313.cern.ch:9197/DataManagement/FileCatalog, dips://lbvobox314.cern.ch:9197/DataManagement/FileCatalog'
```

Reply:
This is needed because LHCb CS has the following structure:

```
DIRAC
{
  ....
  Setups
  {
    LHCb-Production
    {
      ....
      DataManagement = Production
      ....
....

```

Reply:
How is your agent configured? What is the value for the "Setup" parameter?